unittests/beam3/4C_beam3_reissner_test.cpp

```diff
@@ -68,7 +72,199 @@ namespace
+          -0.0327249234748937, 0};
+      A.insert_global_values(17, indices.size(), values.data(), indices.data());
+
+      values = {0, 0.143393430238637, -0.0716967151193184, -0.00238989050397757, 0, 0, 0,
+          0.0130899693899575, -0.00654498469497874, 0, -0.143393430238637, 0.0716967151193184,
+          -0.00238989050397757, 0, 0, 0, 0.0130899693899575, -0.00654498469497874,
+          0.0286786860477280, 0.0238989050397728, 0.0477978100795456};
+      A.insert_global_values(18, indices.size(), values.data(), indices.data());
+
+      values = {-0.143393430238637, 0, -0.358483575596592, 0, -0.00238989050397757, 0,
+          -0.0130899693899575, 0, -0.0327249234748937, 0.143393430238637, 0, 0.358483575596592, 0,
+          -0.00238989050397757, 0, -0.0130899693899575, 0, -0.0327249234748937, 0.0238989050397728,
+          0.143393430238638, -0.00955956201590912};
+      A.insert_global_values(19, indices.size(), values.data(), indices.data());
+
+      values = {0.0716967151193184, 0.358483575596592, 0, 0, 0, -0.00238989050397757,
+          0.00654498469497873, 0.0327249234748937, 0, -0.0716967151193184, -0.358483575596592, 0, 0,
+          0, -0.00238989050397757, 0.00654498469497873, 0.0327249234748937, 0, 0.0477978100795457,
+          -0.00955956201590912, 0.129054087214774};
+      A.insert_global_values(20, indices.size(), values.data(), indices.data());
+    }
+    A.complete();
+
+    int numdof, dimnsp;
+    testele_->element_type().nodal_block_information(testele_.get(), numdof, dimnsp);
+
+    const auto B =
+        Core::FE::compute_null_space(*testdis_, numdof, dimnsp, *testdis_->dof_row_map());
+    Core::LinAlg::MultiVector<double> zero(*testdis_->dof_row_map(), 6);
+
+    A.multiply(false, *B, zero);
+
+    // Check the 1-norm of the nullspace vectors
+    // norm is initialized with ones just to avoid having accidental zeros
+    std::array<double, 6> one_norm{1.0, 1.0, 1.0, 1.0, 1.0, 1.0};
+    zero.Norm1(one_norm.data());
+
+    EXPECT_NEAR(one_norm[0], 0.0, 1e-12);
+    EXPECT_NEAR(one_norm[1], 0.0, 1e-12);
+    EXPECT_NEAR(one_norm[2], 0.0, 1e-12);
+    EXPECT_NEAR(one_norm[3], 0.0, 1e-12);
+    EXPECT_NEAR(one_norm[4], 0.0, 1e-12);
+    EXPECT_NEAR(one_norm[5], 0.0, 1e-12);
+
+    // Check the 2-norm of the nullspace vectors
+    // norm is initialized with ones just to avoid having accidental zeros
+    std::array<double, 6> two_norm{1.0, 1.0, 1.0, 1.0, 1.0, 1.0};
+    zero.Norm2(two_norm.data());
+
+    EXPECT_NEAR(two_norm[0], 0.0, 1e-12);
+    EXPECT_NEAR(two_norm[1], 0.0, 1e-12);
+    EXPECT_NEAR(two_norm[2], 0.0, 1e-12);
+    EXPECT_NEAR(two_norm[3], 0.0, 1e-12);
+    EXPECT_NEAR(two_norm[4], 0.0, 1e-12);
+    EXPECT_NEAR(two_norm[5], 0.0, 1e-12);
+
+    // Check the inf-norm of the nullspace vectors
+    // norm is initialized with ones just to avoid having accidental zeros
+    std::array<double, 6> inf_norm{1.0, 1.0, 1.0, 1.0, 1.0, 1.0};
```

Comment:
Testing for the infnorm should be sufficient. The other tests do not bring additional value. Since they are cheap, we can leave them as they are.